src/ai2_internal/bibentry_predictor_mmda/interface.py

```diff
@@ -30,13 +30,13 @@ class Instance(BaseModel):
     symbols: str
     tokens: List[api.SpanGroup]
     pages: List[api.SpanGroup]
-    bib_entry_boxes: List[api.BoxGroup]
+    bib_entries: List[api.SpanGroup]
```

Comment:
just fyi because I didn't see it in your pr description checklist: this will break the current node config in SPP, since it is configured to send boxgroups.